redis/helm/redis/values.yaml

```diff
@@ -5,12 +5,25 @@ redis:
 
   image:
     registry: gcr.io/pluralsh
-    tag: 6.2.4-debian-10-r0
+    tag: 7.0.5-debian-11-r7
+
+  master:
+    resources:
+      requests:
+        memory: 25Mi
```

Comment:
I feel like these default resources re going to be a good bit too low

Reply:
Do you have any suggestions?